platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminAlertImpl.java

```diff
@@ -75,6 +76,15 @@ public AdminAlertImpl(PersistentStore persistentStore, EventAdmin eventAdmin)
       }
     } catch (PersistenceException pe) {
       LOGGER.debug("Error retrieving system alert.", pe);
+
+      return Collections.singletonList(
+          new Alert(
```

Comment:
❓ What is the effect of returning an alert like this? Will it be clearable by the admin or will that fail also? If the later than it will be confusing that we show an alert but they cannot clear it.

Reply:
As long the alert system cannot contact persistent storage, the alert appears. I think it would be hazardous to allow this alert to be dismissed because critical information may exist, but be unaccessible.